src/Workspaces/Core/Desktop/Workspace/MSBuild/VisualBasic/VisualBasicProjectFileLoader.cs

```diff
@@ -880,7 +880,7 @@ public bool SetLanguageVersion(string languageVersion)
                 {
                     if (!string.IsNullOrWhiteSpace(languageVersion))
                     {
-                        _commandLineArgs.Add("/languageversion:" + languageVersion);
+                        _commandLineArgs.Add("/langversion:" + languageVersion);
```

Comment:
It probably never worked. This only would have impacted customers using MSBuildWorkspace to load their projects, so the scenario is quite small.